src/app/components/Header.tsx

```diff
@@ -0,0 +1,84 @@
+"use client"
+import { Header, PrimaryNav } from "@trussworks/react-uswds";
+import Link from "next/link";
+import Image from "next/image";
+import { usePathname } from "next/navigation";
+import React from "react";
+import styles from "../styles/Home.module.scss";
+import { LinkProps } from "next/link";
+
+interface NavigationLinkProps extends Pick<LinkProps, 'href' | 'onClick'> {
+  children: React.ReactNode;
+}
+
+export default function Navbar() {
+    const basePath = "/dibbs-site";
+
+    const [expanded, setExpanded] = React.useState(false)
+    const onClick = () => { if (window.innerWidth < 1024) setExpanded((prvExpanded) => !prvExpanded) }
+
+    const testItemsMenu = [
+        <Link
+            href='/'
+            key="one"
+            className={`usa-nav__link ${styles.homeNavItem}`}
+            onClick={onClick}>
+
+            <span className={styles.navbarItemText}>Home</span>
+
+        </Link>,
+        (
+          <NavigationLink key="two" href='/our-products' onClick={onClick}>
+            Our products
+          </NavigationLink>
+        ),
+        (
+          <NavigationLink key="two" href='/our-products' onClick={onClick}>
```

Comment:
Should this `href` be `/case-studies` (or something along those lines)? Looks like this could apply to `Footer` as well. Sorry I missed this the first time around 🫣

Reply:
same, fixed